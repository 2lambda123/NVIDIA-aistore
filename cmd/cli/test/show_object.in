echo "0123456789" > $OBJECT_1.txt
ais create bucket $BUCKET_1
ais put $OBJECT_1.txt $BUCKET_1/object_1.txt

ais show object $BUCKET_1/object_2.txt // FAIL "No such object "object_2.txt" in bucket "ais://$BUCKET_1""
ais show object ais://$BUCKET_2/object_1.txt // FAIL "Bucket "ais://$BUCKET_2" does not exist"

ais show object $BUCKET_1/object_1.txt
ais show object $BUCKET_1/object_1.txt --props=size,version
ais show object $BUCKET_1/object_1.txt --props=all

ais show object aiss://$BUCKET_1/object_1.txt // FAIL "invalid backend provider "aiss""
