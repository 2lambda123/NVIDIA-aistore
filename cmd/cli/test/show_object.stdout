PROPERTY         VALUE
^atime(.*)$
checksum         05967d5390ac53b0
name             ais://$BUCKET_1/object_1.txt
size             11B
PROPERTY         VALUE
size             11B
version          1
PROPERTY         VALUE
^atime(.*)$
cached           yes
checksum         05967d5390ac53b0
copies           1
custom           []
ec               -
name             ais://$BUCKET_1/object_1.txt
size             11B
version          1
